metrics/scripts/grafana-provisioning/dashboards/cluster-monitor.json

```diff
@@ -136,7 +136,7 @@
           "hide": false,
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT MODE(last) FROM ( SELECT last(\"leader\") FROM \"$testnet\".\"autogen\".\"replay_stage-new_leader\" WHERE $timeFilter GROUP BY host_id )\n",
+          "query": "SELECT last(\"leader\") FROM \"$testnet\".\"autogen\".\"replay_stage-new_leader\" WHERE $timeFilter ORDER BY \"time\" desc \n",
```

Comment:
Ah, thanks for fixing this! Is there a chance we could keep the "MODE" to ensure one lagging node doesn't influence the metrics?

Reply:
Sorry, I don't follow how `MODE` would work in this case. If a node isn't reporting metrics or hasn't sent their metrics yet even though it is currently the leader, this query will just return the previous leader's id. I don't see how to get more-current data without re-adding the current-leader datapoint from all nodes. What am I missing?

Reply:
Obviously it would be better to order by slot desc, but influxdb only permits order by time.

Reply:
From the linked PR:
> In addition to announcing when they become leader, nodes will also announce when another node becomes leader

So since every node announces the leader, we need to filter out any datapoints that were really recent but came from a delinquent node. `MODE` acts as that filter by using the most common reported leader

Reply:
> Obviously it would be better to order by slot desc, but influxdb only permits order by time.

Exactly, since we can't do that. We can't know if the latest datapoint was for an old slot

Reply:
> So since every node announces the leader, we need to filter out any datapoints that were really recent but came from a delinquent node. `MODE` acts as that filter by using the most common reported leader

The whole point of this PR is that nodes do not announce when another node becomes leader. They only announce when they themselves become leader. So there is only at most one datapoint per slot. If datapoints do share the same time, mode won't show us which is correct.

Reply:
Oh, sorry totally missed this "was removed more than a year ago" part. That sounds good then

Reply:
Awesome, thank you!